common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,222 @@
+@leadership_2 = "gfx/interface/icons/culture_innovations/innovation_leadership_02.dds"
+@raised_banner = "gfx/interface/icons/culture_innovations/innovation_raised_banner.dds"
+@fortifications = "gfx/interface/icons/culture_innovations/innovation_fortifications.dds"
+@siege_weapons = "gfx/interface/icons/culture_innovations/innovation_siege_weapons.dds"
+@levy_building = "gfx/interface/icons/culture_innovations/innovation_levy_building.dds"
+@maa_01 = "gfx/interface/icons/culture_innovations/innovation_maa_01.dds"
+@maa_02 = "gfx/interface/icons/culture_innovations/innovation_maa_02.dds"
+@weapons_and_armor_01 = "gfx/interface/icons/culture_innovations/innovation_weapons_and_armor_01.dds"
+@weapons_and_armor_02 = "gfx/interface/icons/culture_innovations/innovation_weapons_and_armor_02.dds"
+@knight = "gfx/interface/icons/culture_innovations/innovation_knight.dds"
+@majesty_01 = "gfx/interface/icons/culture_innovations/innovation_majesty_01.dds"
+@majesty_02 = "gfx/interface/icons/culture_innovations/innovation_majesty_02.dds"
+@majesty_03 = "gfx/interface/icons/culture_innovations/innovation_majesty_03.dds"
+@nobility_01 = "gfx/interface/icons/culture_innovations/innovation_nobility_01.dds"
+@nobility_02 = "gfx/interface/icons/culture_innovations/innovation_nobility_02.dds"
+@nobility_03 = "gfx/interface/icons/culture_innovations/innovation_nobility_03.dds"
+@nobility_04 = "gfx/interface/icons/culture_innovations/innovation_nobility_04.dds"
+@misc_inventions = "gfx/interface/icons/culture_innovations/innovation_misc_inventions.dds"
+
+@camel = "gfx/interface/icons/culture_innovations/innovation_camel.dds"
+@elephant = "gfx/interface/icons/culture_innovations/innovation_elephant.dds"
+@special_maa_01 = "gfx/interface/icons/culture_innovations/innovation_special_maa_01.dds"
+@special_maa_02 = "gfx/interface/icons/culture_innovations/innovation_special_maa_02.dds"
+
+#culture_group_regional
+####################
+##Tribal##
+####################
+innovation_amphibious = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		OR = {
+			##Nagas
+			this = culture:naga
+			##Murlocs
+			this = culture:murloc
+			this = culture:gorloc
+			##Giblin
+			this = culture:giblin
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -1
+		supply_duration = 0.2
+		no_water_crossing_penalty = yes
+		movement_speed = -0.155
+		supply_duration = -0.25
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		OR = {
+			##Human
+			this = culture:tirassian
+			##Vrykul
+			this = culture:vrykul
+			this = culture:kvaldir
+			this = culture:hyldnir
+			this = culture:frost_vrykul
+			this = culture:drust
+			##Pirates
+			this = culture:pirate_culture
+			this = culture:blackwater
```

Comment:
Zandalari and goblins are navy nations too.
In the good old times, high elves, trolls and ogres were described as navy nations.
https://wow.gamepedia.com/Destroyer
https://wow.gamepedia.com/Juggernaught

Reply:
maybe not goblins since they already have a tech?

Reply:
> maybe not goblins since they already have a tech?

What tech?

Reply:
"Time is money" it's a little lower in the file

Reply:
> "Time is money" it's a little lower in the file

But the goblins don't have naval bonuses having the biggest colonial empire.

Reply:
Why it was resolved?

Reply:
https://wow.gamepedia.com/Trade_Fleets

Reply:
Seafaring has regional conditions in the "regional innovations" branch
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/84
They would get this anyway from the start because of their holdings...unless you want them to specifically be mentioned here.  Just most games I played on here the Trade coalition usually lost all their islands in like 30 years, maybe this will help the AI not die...if you want goblin culture to be coded in here instead of regions lmk

Reply:
> if you want goblin culture to be coded in here instead of regions lmk

Code them in both places. Kezan on the regions and goblins here.